packages/mobile/src/components/AggressiveWarning/AggressiveWarning.component.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import { defaultTheme } from '../../styles/themes/default.theme'
+import { View, Image, StyleSheet } from 'react-native'
+import { Appbar } from '../Appbar/Appbar.component'
+import { Typography } from '../Typography/Typography.component'
+import ExclamationMark from '../../../assets/icons/exclamationMark.png'
+import { AggressiveWarningComponentProps } from '../../screens/AggressiveWarning/AggressiveWarning.types'
+import { Button } from '../Button/Button.component'
+
+const classes = StyleSheet.create({
+  mainWrapper: {
+    padding: 20,
+    alignItems: 'center',
+    marginTop: 10,
+  },
+  image: {
+    width: 96,
+    height: 83,
+  },
+  title: {
+    marginTop: 32,
+    marginBottom: 24,
+  },
+  text1: {
+    lineHeight: 20,
+    textAlign: 'center',
+  },
+  text2: {
+    lineHeight: 20,
+    textAlign: 'center',
+    marginBottom: 16,
+  },
+})
+
+const AggressiveWarningComponent: React.FC<AggressiveWarningComponentProps> = ({
+  handleBackButton,
+  communityName,
+  leaveCommunity,
+}) => {
+  return (
+    <View
+      style={{ flex: 1, backgroundColor: defaultTheme.palette.background.white }}
+      testID={'aggressive-warning-component'}
+    >
+      <Appbar title={'Warning!'} back={handleBackButton} crossBackIcon />
+      <View style={classes.mainWrapper}>
+        <Image source={ExclamationMark} style={classes.image} />
+
+        <Typography fontSize={16} fontWeight={'medium'} style={classes.title}>
+          Possible impersonation attack
+        </Typography>
+
+        <Typography fontSize={14} style={classes.text1}>
+          The owner of{' '}
+          <Typography fontSize={14} fontWeight={'medium'}>
+            {communityName}
+          </Typography>{' '}
+          has registered an invalid username. Either something is very broken, the community owner is trying to
```

Comment:
@holmesworcester should we be more explicit here, e.g. "has registered a duplicate username"?